mesa-dri: enable vc4 driver for armv6l

ericonr opened the pull request:
Related to #20861

xtraeme commented:
missing revbump

ericonr commented:
@xtraeme fixed! Given that xwayland wasn't built for armv6l because of the lack of VC4 driver (at least that's what 85e2e69d439 seems to indicate), can I add it to the xorg-server template and include the commit in this PR?

xtraeme commented:
Sure

ericonr commented:
Given that I'd be enabling a new package to be built, it shouldn't require a revbump, right?

xtraeme commented:
you are wrong, it's the opposite! you need to revbump when the binpkgs change somehow.

ericonr commented:
@xtraeme updated all the necessary packages and added the arch checks. I don't know if there's anywhere I could fit this snippet to avoid duplication, but if there is, it would probably be a good idea. Also added revbumps for everybody, I think.

st3r4g commented on srcpkgs/xorg-server/template:
```diff
@@ -34,7 +34,7 @@ build_options="elogind"
 desc_option_elogind="Rootless Xorg support with elogind"
 
 case "$XBPS_TARGET_MACHINE" in
```
I wonder if this case statement could be removed altogether. All officially supported arches are included, and ppc too. Is this important for some obscure arch?

ericonr replied:
It excludes MIPS, right?

st3r4g replied:
yeah, mips and other arm* (`./xbps-src -h` gives a list)

ericonr replied:
What I meant is that MIPS is also officially supported (at least that's what it says at voidlinux.org), so we have to consider that when building these packages.

travankor replied:
I don't think void builds packages for MIPS... also MIPS as an architecture is going the way of the dodo thanks to RISCV

xtraeme replied:
See http://repo.voidlinux.de by @pullmoll

travankor replied:
Technically, gallium-pipe is hardware agnostic and will support the obscure arches. It's not very efficient, though, but maybe there will be a better software renderer?

ericonr replied:
@xtraeme do you think keeping this check for MIPS is the best way of doing it?

travankor commented on srcpkgs/gdm/template:
```diff
@@ -43,6 +42,11 @@ if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=" glib-devel pam-devel"
 fi
 
+case "$XBPS_TARGET_MACHINE" in
```
missing comment above

ericonr replied:
thanks :)

ericonr commented:
@q66 could you take a look at this PR, please? I rebased it and I think it's as close to ready as it can be, unless we can determine for sure that we can remove the `xorg-server-xwayland` check.

q66 commented:
let's drop the xwayland check, and enable xwayland in xorg everywhere, how about that?
